### PR TITLE
updated readme with instructions for box

### PR DESCRIPTION
**Pull request by alicevhenshaw**
- Added instructions for all included aspects into the readme.
- Removed `README.md` from the `ignore` section in `truffle-box.json` which means the README gets unboxed too - that may people can keep the instructions with them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,100 @@ Included:
...
+A circle CI configuration file is provided already in the repository. This file is set up to perform:
+- `yarn install`
+- test migrations
+- run all tests and calculate their coverage
+- linting (this does not fix linting issues but reports them) 
+- clean up old branches in github
+
+## Migration Utils:
+This tool integrates with truffle migrations to enable a more comprehensive tracking of deployed contracts
+and their addresses. Utilising `migration_utils.js` enables multiple of the same contract to be deployed
```
Comment (poiresel) [inline README.md]: enables the same contract to be deployed multiple times with different settings

File: README.md [hunk 2]
```diff
@@ -12,6 +12,100 @@ Included:
...
+and for these to have different addresses that are both kept track of - which truffle alone does not allow.
+
+## Solidity Coverage
+We use a [branch of solidity-coverage written by leapdao](http://github.com/leapdao/solidity-coverage#master), which enables us to use the newer versions of solidity.
```
Comment (poiresel) [inline README.md]: newer version of solidity (0.5.0+)

File: README.md [hunk 3]
```diff
@@ -12,6 +12,100 @@ Included:
...
+~ pip3 install -r slither-requirements.txt  // to install the necessary requirements for slither
+```
+
+Your local virtual environment is now setup ready to run slither. To run slither on the contracts, how run
+the following:
+```bash
+slither .
+```
+
+## Mythril
```
Comment (poiresel) [inline README.md]: Mythril-Classic

File: README.md [hunk 4]
```diff
@@ -12,6 +12,100 @@ Included:
...
+~ brew install solidity                     // now you have all the necessary dependencies from brew
+
+~ virtualenv venv-mythril                   // to setup a virtual environment in folder ./venv-mythril
+~ source venv-mythril/bin/activate          // to activate your virtual environment
+~ pip3 install -r mythril-requirements.txt  // to install the necessary requirements for mythril
+```
+
+### Now to actually run mythril...
+
+Mythril depends on importing all of the relevant contract imports before analysing the code. Allowing
```
Comment (poiresel) [inline README.md]: Mythril-Classic

File: README.md (hunk 1 above)
Comment (ejwessel) [reply README.md]: Should we provide a 'how to use' the migration_utils.js?

Comment (alicevhenshaw) [reply README.md]: I think that sounds like a good idea Ethan. I'll finish the improvements deepa gave and then do you want to add that? I feel you understand it more than m

Comment (ejwessel) [reply README.md]: yeah i'll add it.

**Pull request merged**